src/APIServer/app.js

```diff
@@ -0,0 +1,59 @@
+const createError = require('http-errors');
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const Promise = require('bluebird'); // eslint-disable-line no-unused-vars
+const mongoose = require('mongoose');
+
+mongoose.Promise = Promise;
+// const viewEngine = require('express-json-views');
+
+// Load app
+const app = express();
+
+// Loading routers
+const noticesRouter = require('./routes/notices');
+
+// Set view engine, JSON (not in use yet)
+// See https://www.npmjs.com/package/express-json-views
+// app.engine('json', viewEngine());
```

Comment:
Are these documentation or old code? If the latter then please remove

Reply:
Revised